homeassistant/components/recorder/websocket_api.py

```diff
@@ -294,6 +296,11 @@ async def ws_adjust_sum_statistics(
     def valid_units(statistics_unit: str | None, display_unit: str | None) -> bool:
         if statistics_unit == display_unit:
             return True
+        if (
+            statistics_unit == DistanceConverter.NORMALIZED_UNIT
+            and display_unit in DistanceConverter.VALID_UNITS
```

Comment:
Can `VALID_UNITS` be a `set` so we don't have to do linear searches every time we do an `in`?

Reply:
Planned for separate PR

Reply:
https://github.com/home-assistant/core/pull/79104